pyproject.toml

```diff
@@ -0,0 +1,109 @@
+[tool.poetry]
+name = "bourbaki-introspection"
```

Comment:
I'm not 100% on this, but `poetry` definitely wants you to only use the `-` separator in the package name, including namespaces.